### DISCUSSION TITLE
how to make xfreerdp run with php on side server?

### DISCUSSION THREAD

**Discussion by buzashop1998** [FreeRDP/FreeRDP#9287] [Q&A]
hello, i need to run this script :
 
 <?php $host = "52.25.33.205"; $user_name = "ben"; $password = "password"; set_time_limit(10); $r = ''; $result = shell_exec("timeout 10s xfreerdp /cert-ignore /v:". $host ." /port:3389 /u:". $user_name ." /p:'". $password ."' +auth-only 2>&1"); if (preg_match_all("/\bAuthentication only, exit status 0\b/",$result, $matches)): echo '1'; elseif(preg_match_all("/\bAuthentication only, exit status 1\b/",$result, $matches)): echo '0';

how to make xfreerdp run with php on side server?

**Comment by akallabeth** [comment]
use `sfreerdp` (the sample client) as `xfreerdp` requires `X11`
